### ISSUE TITLE
Better documentation regarding the use of a custom tab bar component

### ISSUE THREAD

**Issue by svenlombaert** [react-navigation/react-navigation-tabs#117]
Related to: https://github.com/react-navigation/react-navigation-tabs/issues/116

There is little to no documentation about using a custom component as the tab bar with `createBottomTabNavigator`.

Some documentation about the required props + an explanation of what they mean could be very helpful.

**Comment by brentvatne** [comment]
yeah this could be useful. one way this is documented currently is in the themes docs page, notice that we override the bottom tab bar to theme it: https://reactnavigation.org/docs/en/themes.html
